getting_started/step_by_step/scripting_first_script.rst

```diff
@@ -229,6 +229,7 @@ them.
  .. code-tab:: gdscript GDScript
 
     var velocity = Vector2.UP.rotated(rotation) * speed
+
```

Comment:
Might seem superfluous here, but in the next tutorial we replace the `var velocity = ...` line by a multi-line block and add a separation line - it's best to have it from the start IMO.